packages/devtools_app/lib/src/shared/memory/adapted_heap_data.dart

```diff
@@ -5,6 +5,7 @@
 import 'package:collection/collection.dart';
 import 'package:vm_service/vm_service.dart';
 
+import '../../../devtools_app.dart';
```

Comment:
Avoid importing `devtools_app.dart`.

Reply:
addressed in https://github.com/flutter/devtools/pull/5135